### ISSUE TITLE
Remove access control

### ISSUE THREAD

**Issue by bekher** [bekher/officehours#6]
Arbitrary users (i.e., anyone with a Directory ID can queue up; no need to do access control)

**Comment by dkaps125** [comment]
Is this issue so that non-students (i.e., Instructors and TAs) can create tickets?

**Comment by bekher** [comment]
No, this is so we don't need to explicitly add students to a class. Meaning, anyone with CAS authentication can log in to the app as a student. If their account isn't in the database, we will create one for them, send them to a "fill out your profile" page where they will fill out their first name, choose a course they're trying to attend OH for (when we have multiple courses available in one instance) and then redirect them to the proper course page.

**Comment by bekher** [comment]
Will work on this when handling #37 since we will want to revamp the login system when adding multiple courses to the app.
